### ISSUE TITLE
Twofactor settings using the multi select is a bit confusing

### ISSUE THREAD

**Issue by nickvergessen** [nextcloud/server#14403]
1. When you click into the field, there are no options. You first have to type something so the search starts triggering.
2. If you search slowly (char by char) and wait for the ajax options to return they are duplicated. `_.sortedUniq` seems to not deduplicate, replacing it with `_.uniq` atleast makes them unique again, but now the sorting is "random".
3. If you searched for a group, didn't find it, and press `backspace` too long, it also deselects all previous selected groups. I don't think this is what most users would expect.

cc @nextcloud/javascript

**Comment by rullzer** [comment]
@skjnldsv @ChristophWurst seems 3 is still an issue.
But I guess this is a multiselect thing we should solve in general?

**Comment by ChristophWurst** [comment]
Yes, this is (likely, haven't checked) coming from the upstream multiselect component. I don't know if we can or want to prevent this by default or should add a property to disable if necessary.

**Comment by szaimen** [comment]
I cannot reproduce this because I don't find the "field" you are talking about. Could you please share a screenshot?
Thanks! :)

**Comment by ChristophWurst** [comment]
Joas is talking about the admin two-factor settings field about included/excluded groups. Did you check the user two-factor settings perhaps?

**Comment by ChristophWurst** [comment]
There was also https://github.com/nextcloud/server/pull/14405 and it might make this ticket obsolete.

**Comment by szaimen** [comment]
Ah thanks!
Yes, except the 3rd point everything was fixed.
But maybe we should close this for now and open a new issue if the 3rd point comes up again?
